tests/PHPStan/Analyser/data/bug-8442.php

```diff
@@ -0,0 +1,9 @@
+<?php declare(strict_types = 1);
+
+namespace Bug8442;
+
+use function PHPStan\Testing\assertType;
+use DateInterval;
+
+assertType('false', DateInterval::createFromDateString('foo'));
+assertType('DateInterval', DateInterval::createFromDateString('1 Day'));
```

Comment:
please wrap this code into a function or method.

that way we can make sure that this code is not evaluated while phpstan is scanning the codebase

Reply:
done